3.0-RELEASE-NOTES.md

```diff
@@ -62,4 +62,11 @@ methods: {
 }
 ```
 
-Please see [related code change](https://github.com/strongloop/loopback/pull/2174) here.
\ No newline at end of file
+Please see [related code change](https://github.com/strongloop/loopback/pull/2174) here.
+
+## Removing unused user properties
+- credentials
+- challenges
+- status
+- created
+- lastUpdated
```

Comment:
Please add a bit more meat to the release note. Explain how to upgrade apps relying on these properties - users should redefine them in their model JSON files, show an example of such JSON definition.

Also add a link to this pull request, to make it easy to find discussions that lead to the change.